docs/.vuepress/enhanceApp.js

```diff
@@ -10,7 +10,7 @@ export default ({
   router, // the router instance for the app
   siteData // site metadata
 }) => {
-
+/**
```

Comment:
These are Asgardeo specific even if they are commented out. Consider removing them.

```
'/guides/your-apk/manage-organizations/': '/guides/organization-management/manage-organizations/',
      '/guides/your-apk/manage-environments/': '/guides/organization-management/manage-environments/',
      '/guides/your-apk/delete-organizations/': '/guides/organization-management/manage-organizations/#delete-organizations',
      '/references/apk-events/': '/guides/apk-events/',
      '/sdks/': '/integrations/',
      '/get-started/try-your-own-app/angular/': '/get-started/try-your-own-app/',
      '/get-started/try-samples/qsg-spa-angular/': '/get-started/try-samples/'
     ```

Reply:
Fixed in https://github.com/wso2/docs-apk/pull/22/commits/6ac0155e115bf8c55b9ab6bf4c5345bed5da097e